README.md

```diff
@@ -44,12 +39,11 @@ Dependencies
 
 The following dependencies are required to run Procrustes properly,
 
-* Python >= 3.6: http://www.python.org/
-* NumPy >= 1.21.5: http://www.numpy.org/
-* SciPy >= 1.5.0: http://www.scipy.org/
-* PyTest >= 5.3.4: https://docs.pytest.org/
-* PyTest-Cov >= 2.8.0: https://pypi.org/project/pytest-cov/
-
+* Python >= 3.6: <http://www.python.org/>
+* NumPy >= 1.21.5: <http://www.numpy.org/>
+* SciPy >= 1.5.0: <http://www.scipy.org/>
+* PyTest >= 5.3.4: <https://docs.pytest.org/>
+* PyTest-Cov >= 2.8.0: <https://pypi.org/project/pytest-cov/>
```

Comment:
GitHub takes care of rendering the urls automatically, https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls. Can you explain why you do this? Thank you.

Reply:
I didn't. Just the result of auto-formatting on save from [`markdownlint`](https://github.com/DavidAnson/markdownlint). The reason it does this (I assume) is that auto-converting URLs into HTML anchor tags is not part of the markdown spec. I.e. not all markdown parsers do this, GFM is just one variant that does.

Reply:
This is a generalized solution. I am going to merge it soon. Thanks for the contribution! @janosh